src/mount.rs

```diff
@@ -49,6 +49,9 @@ pub fn mount_root(options: &CmdlineOptions) -> Result<()> {
         return Err("root= not found in /proc/cmdline".into());
     }
 
+    let root_device = options.root.as_ref().ok_or("No root device argument")?;
+    wait_for_device(root_device)?;
```

Comment:
Hmmm, I'm not sure if we can do this unconditionally like this. e.g. nfsroot has not root device.